parsons/databases/mysql/create_table.py

```diff
@@ -0,0 +1,159 @@
+import ast
+import petl
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MySQLCreateTable():
+
+    def __init__(self):
+
+        pass
+
+    def data_type(self, val, current_type):
+        # Evaluate the MySQL data type of a given value. Then return
+        # the data type of the value, while also weighing what the
+        # current type is.
+
+        # Stop if the current type is already a varchar
+        if current_type == 'varchar':
+            return current_type
+
+        # Convert to string to reevaluate data type
+        try:
+            t = ast.literal_eval(str(val))
+        except ValueError:
+            return 'varchar'
+        except SyntaxError:
+            return 'varchar'
+
+        # If a float, then end
+        if type(t) == float:
+            return 'float'
+
+        # If an int and current type is not a float, then determine type of int.
+        if type(t) == int and current_type != 'float':
+            # Make sure that it is a valid integer
+            if not self.is_valid_integer(val):
+                return 'varchar'
+            # Use smallest possible int type above TINYINT
+            if (-32768 < t < 32767) and current_type not in ['int', 'bigint', 'mediumint']:
+                return 'smallint'
+            elif (-8388608 < t < 8388607) and current_type not in ['int', 'bigint']:
+                return 'mediumint'
+            elif (-2147483648 < t < 2147483647) and current_type not in ['bigint']:
```

Comment:
Is there a system function call we could draw these boundary values from, rather than hardcoding them?